server/src/generated/prisma-client/index.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+var prisma_lib_1 = require("prisma-client-lib");
```

Comment:
Can you replace var for es6?

Reply:
`server/generated` - all code here is auto-generated by prisma and you not able to controll it